CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 ## [unreleased]
+## Features
+- [#353](https://github.com/influxdata/influxdb-client-go/pull/353) Synced generated server API with the latest [oss.yml](https://github.com/influxdata/openapi/blob/master/contracts/oss.yml). Simplified generated client stub.
+
+### Breaking change
+- [#353](https://github.com/influxdata/influxdb-client-go/pull/353):
+  - Interface `Client` has been extended with `APIClient()` function.
+  - Simplified methods of generated server API client stub:
+    - Each server call has a single function with all the necessary code
+    - Each server call function takes all arguments in single the wrapper struct
+    - Functions return directly type, instead of a response wrapper struct. 
```

Comment:
It would be better to describe what exactly changed. This current description is hard to understand. For example:
```suggestion
  - The generated client API changed
    - Function names are simplified (was `PostDBRPWithResponse`, now `PostDBRP`) 
    - All functions now accept a context and a simgle wrapper structure with request body and HTTP parameters
    - The functions return deserialized response body or an error (it was a response wrapper with a status code that had to be validated by the client)
```